ios/FluentUI.Demo/FluentUI.Demo/Demos/PeoplePickerDemoController.swift

```diff
@@ -87,8 +90,8 @@ class PeoplePickerDemoController: DemoController {
         peoplePicker.showsSearchDirectoryButton = variant.showsSearchDirectoryButton
         peoplePicker.numberOfLines = variant.numberOfLines
         peoplePicker.allowsPickedPersonasToAppearAsSuggested = variant.allowsPickedPersonasToAppearAsSuggested
-        peoplePicker.showsSearchDirectoryButton = variant.showsSearchDirectoryButton
```

Comment:
sorry why are we deleting this?

Reply:
I noticed it was a duplicated line of code (line 87/90)